features/step_definitions/aruba_steps.rb

```diff
@@ -6,3 +6,13 @@
 Given 'I ignore previous output' do
   aruba.command_monitor.clear
 end
+
+Then(/^(?:the )?output should( not)? contain "([^"]*)" once$/) do |negated, expected|
```

Comment:
Please do not add support for "not". Firstly, you don't need it. Secondly, it is ambiguous. `The output should not contain "hello" once` sounds like "should not occur anywhere", but your implementation would also be green with two or more occurrences.